doc/user/OPTIONS.md

```diff
@@ -23,6 +23,7 @@ Options|Default|Description
 \-\-dry-run||Do not read any configuration file and consider only the command line options.
 \-\-no-render||Print information about the first provided file (as with \-\-verbose) and exit, without rendering anything, useful to recover information about a file.
 \-\-max-size=\<size in MiB\>|-1|Prevent F3D to load a file bigger than the provided size in Mib, -1 means unlimited, useful for thumbnails.
+\-\-load-plugins=\<paths or names\>||List of plugins to load separated with a comma.
```

Comment:
doc is a bit lacking I think, should specify that it can be a name or a path

Reply:
It's literally written `\<paths or names\>` :)